Sources/AllOfSchema.swift

```diff
@@ -3,6 +3,14 @@ import ObjectMapper
 public struct AllOfSchema {
     public let metadata: Metadata
     public let subschemas: [Schema]
+    
+    /// Determines whether or not the schema should be considered abstract code
+    /// generation purposes. This can be used to indicate that a schema is an
+    /// interface rather than a concrete model object.
+    ///
+    /// Corresponds to the boolean value for `x-abstract`. The default value is
+    /// false.
+    public let abstract: Bool
 }
 
 public struct AllOfSchemaBuilder {
```

Comment:
I didn't catch this is my other code review, but the builders should not be public. The builders are used by the library internally to map + validate the document, and then build recursively generates the finished objects which *are* public.